waltid-services/waltid-issuer-api/src/main/kotlin/id/walt/issuer/issuance/IssuerApi.kt

```diff
@@ -221,26 +237,61 @@ fun Application.issuerApi() {
                                     )
                                 }
                             }
+                            "400" to {
+                                description = "Bad request"
+                            }
                         }
                     }) {
-
                         val body = context.receive<Map<String, JsonElement>>()
+                        val keyJson = body["issuerKey"]?.jsonObject
+                            ?: return@post context.respond(
+                                HttpStatusCode.BadRequest,
+                                "Missing issuerKey in the request body."
```

Comment:
Look how this message is much more specific than the one at line 121 ("Issuer key must not be empty")